src/apps/vpn/addons/state/addonsessionstate.h

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef ADDONSESSIONSTATE_H
+#define ADDONSESSIONSTATE_H
+
+#include <QHash>
+#include <QJsonValue>
+#include <QObject>
+#include <QString>
```

Comment:
None of these headers are needed.

Reply:
How do you know this? I always wonder haha